Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.3
+FROM alpine:3.18.0
```

Comment:
This will save you some maintenance going forward.

```suggestion
FROM alpine:3.18
```

Reply:
Thanks!